metaswitch/common/generate_stats_csv.py

```diff
@@ -153,12 +153,31 @@ def parse_mib_file(path):
     mib_file = mib.MibFile(path)
 
     def stat_test(stat):
-        """Filter for leaf nodes in OID tree."""
-        # TODO: pick the right nodes!
+        """Filter for statistics nodes - they're the leaf nodes in a table
+        which aren't indices."""

```

Comment:
Docstrings should generally be:
* A one-line pithy summary.
* A blank line.
* The rest of the text.